app/views/layouts/casebooks.html.erb

```diff
@@ -41,6 +41,10 @@
     <div class="content">
       <div class="casebook-inner">
         <div class="top-strip"></div>
+        <% if @casebook.present? %>
+          <% content_for :custom_skip_target do %>i'm-arbitrary-text<% end %>
```

Comment:
Why are you using content_for?

Reply:
This is a trick for letting a "child" view overrule something in a parent view/layout. In the layout, a default skip target is provided unless a child/grandchild/etc. has provided content for custom_skip_target (https://github.com/harvard-lil/h2o/pull/284/files#diff-e3a4a6249a8f8ae3c560f0ba99f7134bR26). So, to customize the location of the skip link, you provide some arbitrary content in the child, and then put a skip target in your customized spot. It'd be nice if it could be empty, rather than having some arbitrary text, but it looks like that doesn't work in Rails.